compose.yaml

```diff
@@ -66,7 +66,7 @@ services:
       - worker
       - scheduler
     environment:
-      CYPRESS_baseUrl: http://server:5000
+      CYPRESS_baseUrl: http://server:5001
```

Comment:
no need to make this change. cypress will be using docker network and in docker network redash server is on port 5000

Reply:
Thanks.  I'll push a follow up commit in a few minutes.  Just want to see if the Cypress tests get further with the current commit first.  In my local testing they're now passing. :smile:

Reply:
Pushed a commit to change that back to 5000.